dist/redhat/scylla.spec.mustache

```diff
@@ -77,7 +77,7 @@ getent passwd scylla || /usr/sbin/useradd -g scylla -s /sbin/nologin -r -d %{_sh
 %post server
 # Upgrade coredump settings
 if [ -f /etc/systemd/coredump.conf ];then
-    /opt/scylladb/scripts/scylla_coredump_setup
+    /opt/scylladb/scripts/scylla_coredump_setup --skip-test
```

Comment:
Maybe we shouldn't run it at all from rpm/deb, and leave it to scylla_setup.

Reply:
@avikivity dropped scylla_coredump_setup from scylla.spec, also drop --skip-test option.